.github/workflows/develop_freeze_comment.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will comment on pull tequests that are submitted while develop
+# is frozen during the week of release. Skips any pull tequests that have the
+# label 'ignore-freeze'.
+# This workflow should be enabled only while develop is frozen.
+
+name: Develop Freeze PR Comment
+
+on:
+  # Trigger the workflow on pull request on develop branch
+  pull_request:
+    types:
+      - opened
+      - reopened
+    branches:
+      - develop
+
+jobs:
+  freeze-comment:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'ignore-freeze') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR Comment
+        uses: github-actions-up-and-running/pr-comment@v1.0.1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message: "Thanks for the PR! The develop branch is currently frozen in preparation for the release so it may not be addressed until after release week."
```

Comment:
For testing purposes, I was thinking it might make sense to change this message and then change it back to this one after we confirm it works right.